ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_path_selection.py

```diff
@@ -67,6 +66,8 @@ def _autovpn_policies(self) -> list:
                     }
                 )
                 rule_id_offset = 1
+                # Eurk
+                policy["name"] = policy["original_name"]
```

Comment:
This is in-place updating the policy name of _filtered_wan_policies, so it becomes even harder to track what is going on here.

Reply:
hmmm weird my rebase must have failed

Reply:
Fixing in next commit actually - but this was safe given that this is only used in one place and I copied the default (though easy to break in a refactoring)